### ISSUE TITLE
Request for Left margin in Note Preview Panel

### ISSUE THREAD

**Issue by AlbusPercivalDumbledore** [pbek/QOwnNotes#2491]
Hi, @pbek! Could you please increase the left margin in Note-Preview panel, and make it consistent with Note-Edit panel? Text starts right on the edge of Note-Preview panel, at present.

![pic](https://user-images.githubusercontent.com/82368617/157067833-1980a8ed-75e9-42a4-a395-03783f59bec7.png)

**Comment by pbek** [comment]
No, sorry, I can't really do that. It would break too much for other users and use cases. And also the editor is configurable the other way around, by setting how many characters are shown in one line (approximately). You can even set a custom value for that. Or you can play around with the padding of the preview, there is a script for that.
